app/views/devise/sessions/new.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="col px-0">
-  <div class='card card-cas'>
-    <div class='card-header text-white bg-primary'>
-      <%= content_tag :h2, t('blacklight.login.netid_header'), class: 'card-title' %>
-    </div>
+  <div class='card border-0'>
     <div class='card-body text-center'>
-      <%= link_to t('blacklight.login.netid_button'), user_cas_omniauth_authorize_path, :class => 'btn btn-primary' %>
-      <p class="card-text"><%=t('blacklight.login.redirect_text')%></p>
-      <p><a role="button" data-toggle="collapse" href="#collapseBarcode" aria-expanded="false" aria-controls="collapseBarcode">
-        <%=t('blacklight.login.barcode_text')%>
+      <%= link_to t('blacklight.login.netid_login_msg'), user_cas_omniauth_authorize_path, :class => 'btn btn-primary' %>
+      <hr>
+      <p class="or-divider">
```

Comment:
Can you make sure that the spacing for the or divider is consistent with the requests page? The line appears too close to the blue login button.